### DISCUSSION TITLE
Renovate runs pipeline multiple times for the same MR (for the same commit, but different hash)

### DISCUSSION THREAD

**Discussion by wojcikm** [renovatebot/renovate#23933] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

Looks like missing configuration option.

### Wanted end result.

Do not trigger pipelines in open MR's multiple times.

### What you tried so far.

Search for config options - not found any applicable. 

I want to limit those runs, preferably to single one, only during MR creation. Screenshot for reference.

<img width="719" alt="image" src="https://github.com/renovatebot/renovate/assets/19979741/dbd5ec6b-7136-42d8-839b-6edbd651ce00">

**Comment by rarkins** [comment]
That does not look like the same commit to me - each line I looked at had a different commit:

![image](https://github.com/renovatebot/renovate/assets/6311784/61676a53-342f-4af6-a46c-de61e93c724e)
